### PR TITLE
feat: add description to address

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: packages/neuron-wallet/src/addresses/entities/address.ts [hunk 1]
```diff
@@ -49,6 +49,12 @@ export default class Address extends BaseEntity {
   })
   version!: AddressVersion
 
+  @Column({
+    type: 'text',
```
Comment (classicalliu) [inline packages/neuron-wallet/src/addresses/entities/address.ts]: Does this description will be long ? Should I use `text` or `varchar` ? @ashchan

Comment (ashchan) [reply packages/neuron-wallet/src/addresses/entities/address.ts]: Limit to a max length is reasonable.

Comment (classicalliu) [reply packages/neuron-wallet/src/addresses/entities/address.ts]: `varchar` is limited to 255 chars and text is much longer, so should we change to varchar to have a limit ?

Comment (ashchan) [reply packages/neuron-wallet/src/addresses/entities/address.ts]: I believe `varchar` is enough. On UI or app layer, we can limit that to 100 UTF8 characters.

Comment (classicalliu) [reply packages/neuron-wallet/src/addresses/entities/address.ts]: updated to `varchar`

**Pull request merged**